### DISCUSSION TITLE
please help.

### DISCUSSION THREAD

**Discussion by ppg00** [vercel/community#3440] [Help]
### Summary

Hello,

I have deploy on Vercel via Github App.
But deploy with status Queued in 1 day.

Please help. Thanks.

### Steps to Reproduce

![chrome_screenshot_31 Jul 2023 16_00_38 GMT+01_00](https://github.com/vercel/community/assets/133581531/65bf7b91-c71b-45b9-918d-516b50738260)

**Comment by amyegan** [comment]
Hey @ppg00. If you share the deployment id or the unique url of a recent successful deployment, then I can look into this for you 🙏

**Reply by toniengelhardt** [reply]
Hey @amyegan, I have the same problem. All my builds get stuck in queued since two days and there are no builds running. First the builds did not complete and timed out and after trying to debug for a while they do not execute at all anymore now. Is there a way to cancel or reset all builds via CLI or smth? 

User ID: hbLowZzQJBHbf00ooYal4jEy 

One of the builds that's stuck: https://vercel.com/toniengelhardt/promptmetheus-website/G1YqZsqqSV6WUG1KMyXgzhLpSA3A 

Thanks 🙏🏽

**Reply by amyegan** [reply]
Hey there, I responded in https://github.com/vercel/community/discussions/539#discussioncomment-6948821
